internal/resources/result.go

```diff
@@ -31,6 +31,11 @@ func (r Result) Requeue() bool {
 	return r.requeue
 }
 
+// Yield returns true if current processing should be discontinued.
+func (r Result) Yield() bool {
+	return r.requeue || r.err != nil
```

Comment:
Should we use result.Requeue() here instead?

Reply:
Requeue returns the value of requeue. It's a public accessor method for that field. So this direct use of the value is just "cutting out the middle man".  If Requeue() became fancy in the future and was the product of multiple inputs we could simply upate Yield as well.

Reply:
update Yield as well.... to use Requeue() then.